react/week4/src/components/MyRepos.js

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from "react";
+
+function MyRepos() {
+  const [repo, setRepo] = useState([]);
+  useEffect(() => {
+    const MY_URL = `https://api.github.com/users/MirSzabo/repos`;

```

Comment:
We should only use template stings ` `` ` only when we want to pass different values in the string.
```suggestion
    const MY_URL = "https://api.github.com/users/MirSzabo/repos";

```